src/components/MiniMap/MiniMap.tsx

```diff
@@ -28,71 +19,81 @@ interface MiniMapProperties {
   setMapOption: (input: string) => {};
 }
 
+const getCounty = (stateId: string, fullFips: string) =>
+  _find(
+    // @ts-ignore: US_STATE_DATASET is .js, but this is valid
+    US_STATE_DATASET.state_county_map_dataset[stateId].county_dataset,
+    ['full_fips_code', fullFips],
+  );
+
 /**
  * TODO(sgoldblatt): Don't pass in the setState vars here and use a provider
  * to set the selected MobileMenu and map options as well
  */
-const MiniMap: FunctionComponent<MiniMapProperties> = (
-  props: MiniMapProperties,
-) => {
+const MiniMap: FunctionComponent<MiniMapProperties> = ({
+  projections,
```

Comment:
I kept the existing API, but it looks like the projections are only used to get the state name, I can probably get that from a JSON file.